### PR TITLE
K8s docs changes for 1.28

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Added 1.28 components, upgrading and release notes page
- added main release notes
- updated metallb docs
- updated navigation
-  updated release notes for several bugfix updates of prior releases 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- A reminder - ALL these changes have already been reviewed and approved by the K8s team - we aren't expecting you to read all the content, just check there is nothing obvious wrong with the new/changed pages e.g. misrendering.

### CODE REVIEW

**Comment by evilnick** [comment]
NB - the woke check is always going to fail on these pages. We do what we can but some of them are external URLS

File: templates/kubernetes/docs/install-offline.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ is covered in the [proxy documentation][].
 
 
 <!-- LINKS -->
-[LXD-image]: https://documentation.ubuntu.com/lxd/en/latest/image-handling
+[LXD-image]: https://linuxcontainers.org/lxd/docs/master/image-handling
```
Comment (anthonydillon) [inline templates/kubernetes/docs/install-offline.md:261]: Looks like the docs have moved to documentation.ubuntu.com

File: templates/kubernetes/docs/install-local.md [hunk 2]
```diff
@@ -247,7 +247,7 @@ juju config kubernetes-worker sysctl="{ fs.inotify.max_user_watches=1048576 }"
 
 <!-- LINKS -->
 
-[lxd-home]: https://ubuntu.com/lxd
+[lxd-home]: https://linuxcontainers.org/
```
Comment (anthonydillon) [inline templates/kubernetes/docs/install-local.md:250]: Isn't the ubuntu.com lint the main project homepage now?

File: templates/kubernetes/docs/packages.md [hunk 3]
```diff
@@ -33,6 +33,16 @@ These are installed in the same way. For example:
 sudo snap install kubectl --classic
 ```
 
+<div class="p-notification--information is-inline">
+  <div class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <p class="p-notification__message">Note: If you install tools such as **kubectl** from both the snap
```
Comment (anthonydillon) [inline templates/kubernetes/docs/packages.md:39]:
```suggestion
    <p class="p-notification__message">If you install tools such as **kubectl** from both the snap
```

**Review by anthonydillon — APPROVED** [review]
LGTM with a few comments in the code

File: templates/kubernetes/docs/install-offline.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/install-offline.md]: thanks for catching these

**Pull request merged**